Header header
geometry_msgs/Point[] points
float64[] info_gains